src/view/TodoPage.tsx

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { InterfaceContent } from './styles';
 import Title from '../Components/Title';
 import Input from '../Components/Input';
+import { ITodo } from './types';
+import Card from '../Components/Card';
 
 const TodoPage = () => {
+const [todo, setTodo] = useState('')
+const [todoList, setTodoList] = useState<ITodo[]>([])
+
+
+const handleSubmit = (e: React.FormEvent) => {
+  e.preventDefault()
+  setTodoList([...todoList, {id: Date.now(), todo:todo, isFinished:false}])
+  setTodo('')
+  console.log(todoList)
```

Comment:
Should this console be here?

Reply:
I was only using it for tests purposes, thanks Mateus!